### DISCUSSION TITLE
how can i change image modal title?

### DISCUSSION THREAD

**Discussion by AndreeaCoaja** [GrapesJS/grapesjs#5836] [Q&A]
Hi! I have looked through this doc: https://github.com/GrapesJS/grapesjs/blob/master/src/asset_manager/config/config.ts and i could not find any way I can change the title from image modal:
![image](https://github.com/GrapesJS/grapesjs/assets/53050097/9c122aee-22ad-4caf-bc28-a659f2578eaa)
I have tried like this so far, but it didn't work:
![image](https://github.com/GrapesJS/grapesjs/assets/53050097/c2ea6827-a515-4ba5-a05a-addb31ec3172)
Is there any easy way I can change it?

Another question is regarding to uploading of files different than images. I did a function where as return an array with data is sent as in documentation:
![image](https://github.com/GrapesJS/grapesjs/assets/53050097/4489b2c7-f643-4da4-8af8-221c667f5045)
The problem is that in pages array from json (after this array is sent), the type of the file is still image. While the type of the file in assets array is correct: 
![image](https://github.com/GrapesJS/grapesjs/assets/53050097/d98b568a-86bf-44dd-a277-50969cd357ba)
How can i change this? And is it possible to diplay the icon of a file different if it's not an image?
![image](https://github.com/GrapesJS/grapesjs/assets/53050097/c84db1d4-54b0-4ae8-be86-b2646d442b40)

Thank you!

**Comment by artf** [comment]
The title of the asset manager modal can be changed via the [i18n](https://grapesjs.com/docs/modules/I18n.html) module (here is the string [reference](https://github.com/GrapesJS/grapesjs/blob/717e4297ce3010866af0b14976755bae3e929540/src/i18n/locale/en.js#L4-L9)).

The type of the component has nothing to do with the type of the asset. You should create the appropriate component to handle your assets properly.
